scripts/opx_build

```diff
@@ -53,6 +53,10 @@ repos=( opx-logging
 	opx-tmpctl
 	)
 
+if [ ! -z "$1" ] && [ "$1" != "all" ]; then
```

Comment:
The script is renamed in this PR: "opx_build_all" -> "opx_build"

Should I revert to original name?